archive/tarfile.go

```diff
@@ -0,0 +1,67 @@
+package archive
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	V "github.com/gspivey/drayage/volume"
+)
+
+//WriteToTar tars any found volumes
+func WriteToTar() (err error) {
+	volumes, err := V.ListVolumes()
+	for _, vol := range volumes {
+		dir, err := os.Open(vol.Mountpoint)
+		if err != nil {
+			return fmt.Errorf("can't open %v (did you use sudo?)", vol.Name)
```

Comment:
On your return "chain" the errors like in the example below:

```go
if err := dec.Decode(&val); err != nil {
    if serr, ok := err.(*json.SyntaxError); ok {
        line, col := findLine(f, serr.Offset)
        return fmt.Errorf("%s:%d:%d: %v", f.Name(), line, col, err)
    }
    return err
}
```

Something like this should work:

```go 
return fmt.Errorf("can't open %v (did you use sudo?) : %v ", vol.Name, err)
```